package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@zaproxy/front-end-tracker",
-  "version": "1.0.0",
+  "version": "1.0.1",
```

Comment:
Does it worth updating the version if 1.0.0 was not yet released? (or was it?)

Reply:
It has been, under [my personal npm account](https://www.npmjs.com/package/@pamplemousse/front-end-tracker).

On my own repo, I need to do this to be able to publish it (npm will not allow me to re-publish under the same version).

And from there, I will be able to update the [client-side javascript code of the FrontEndScanner addon](https://github.com/Pamplemousse/zap-extensions/blob/master/src/org/zaproxy/zap/extension/frontEndScanner/files/frontEndScanner/front-end-scanner.js#L13) to require the new version.

I am pushing it here because the sooner we do not reference my personal fork, the better it is.
And I would prefer to keep consistency between the versions in two repos.

P.S.
  * Publishing on zaproxy organization on npm is not something that I can do.
  * I have some work going on to get travis to publish on npm.

I plan on chatting about this process with @psiinon later this week.